hugo-toha/toha
# Embed pdf

### Issue
[Adding PDF Shortcut #522](https://github.com/hugo-toha/toha/issues/522)

### Description

Adds the latest version of pdf-js (from: https://mozilla.github.io/pdf.js/) to the static/js folder, and the embed-pdf shortcode (maybe could be renamed to pdf?) to layouts/shortcodes. This shorcode was mainly taken from [anvithks/hugo-embed-pdf-shortcode](https://github.com/anvithks/hugo-embed-pdf-shortcode), as is implied to preserve copyright. Maybe an issue should be opened in his repo to let him now? (not necessary but nice)

### Test Evidence

A minimal reproducible example can be reproduced like so:

1. Create a test post in toha/content/posts/test/
2. Fill index.md with:
```
---
---

{{< embed-pdf src="/posts/test/test.pdf" >}}
```
3. Add a test pdf in toha/content/posts/test/test.pdf
4. Issue: `hugo serve`

The result should be like so:

![Screenshot 2022-10-18 at 12-04-00 Screenshot](https://user-images.githubusercontent.com/48572431/196400812-e94f2fab-58c3-4938-b7c0-bf443a8da870.png)
